test/regress/regress1/nl/iand-native-2.smt2

```diff
@@ -0,0 +1,18 @@
+; COMMAND-LINE: --iand-mode=value
+; COMMAND-LINE: --iand-mode=sum --bvand-integer-granularity=1
```

Comment:
I think some of these large amounts of variants seem a bit excessive, do we need to test for multiple granularities here?  I'd prefer if only a small number of benchmarks varied to the integer granularity.